commit ae8cb8c3b63365f3878aa887646b5da01638c276
Author: Tom Hendrikx <tom@whyscream.net>
Date:   Wed Feb 6 13:52:04 2013 +0100

    Revert "Enable strip again"
    
    This reverts commit 52acad4b66ff6fb11f09957b73f437038bd081be.

diff --git a/GNUmakefile.in b/GNUmakefile.in
index 43dff9b..6196303 100644
--- a/GNUmakefile.in
+++ b/GNUmakefile.in
@@ -38,10 +38,10 @@ INSTALL_DATA=$(INSTALL) -m 644
 	${GCC} ${CFLAGS} ${GLIB_CFLAGS} -c $<
 
 ifeq (${ARCHIVE_L},no)
-all:	rdup rdup-up strip
+all:	rdup rdup-up
 	@echo "WARNING: ** No archive library found; not building rdup-tr"
 else
-all:	rdup rdup-up rdup-tr strip
+all:	rdup rdup-up rdup-tr
 endif
 	@chmod +x ${SH}
 	@if [ "${NETTLE_L}" = "no" ]; then echo "WARNING: ** No nettle library found; rdup-tr has no encryption"; fi
